src/alexandria3k/data_sources/crossref.py

```diff
@@ -18,18 +18,15 @@
 #
 """Crossref publication data"""
 
-import abc
-
 from alexandria3k.data_source import (
-    ROWID_INDEX,
     DataFiles,
     DataSource,
-    ElementsCursor,
     FilesCursor,
     StreamingCachedContainerTable,
 )
 from alexandria3k.db_schema import ColumnMeta, TableMeta
 from alexandria3k.file_cache import get_file_cache
+from alexandria3k.json_source import JSONElementsCursor, RecordsCursor
```

Comment:
We use the convention of capitalizing only the first character of acronyms.  So this would be  `JsonElementsCursor`. But see following comments.